pkg/auth/agent_local_authenticator_test.go

```diff
@@ -6,24 +6,45 @@ import (
 	"strings"
 	"time"
 
-	"github.com/go-openapi/strfmt"
 	"github.com/golang-jwt/jwt/v4"
-	"github.com/google/uuid"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	"github.com/openshift/assisted-service/internal/common"
 	"github.com/openshift/assisted-service/internal/gencrypto"
 	"github.com/sirupsen/logrus"
 )
 
+func generateToken(privateKkeyPem string, expiry ...time.Time) (string, error) {
```

Comment:
Typo
```suggestion
func generateToken(privateKeyPem string, expiry ...time.Time) (string, error) {
```

Reply:
Are you ever going to provide multiple `expiry` values?
I mean, this works but you could also use a pointer to indicate present or not instead of a list.